core/server/web/api/v2/admin/middleware.js

```diff
@@ -1,7 +1,27 @@
-const prettyURLs = require('../../../middleware/pretty-urls'),
+const url = require('url'),
+    debug = require('ghost-ignition').debug('api'),
+    prettyURLs = require('../../../middleware/pretty-urls'),
     cors = require('../../../middleware/api/cors'),
-    urlRedirects = require('../../../middleware/url-redirects'),
-    auth = require('../../../../services/auth');
+    {getAdminRedirectUrl} = require('../../../middleware/url-redirects'),
+    auth = require('../../../../services/auth'),
+    urlService = require('../../../../services/url');
+
+const urlRedirects = (req, res, next) => {
```

Comment:
Not sure this should live here 🤔 This file in general should die IMO.
What was your thinking? :)

Reply:
For now, this is kind of the best place to put it. Was thinking we can move all things together during 'middleware refactor' step.

Reply:
But also up to kill this file now :smile:

Reply:
Okay i am fine with keeping it as it is now :)
@rishabhgrg is working on 

> Refactor web/middleware → web/shared